troubleshooting/faq.md

```diff
@@ -6,6 +6,48 @@ sidebar: sidebars/troubleshooting.html
 
 # Frequently Asked Questions
 
+### Can I use pulp-admin with katello?
+
+While it is typically not encouraged, you can use pulp-admin with katello for troubleshooting purposes.
+
+1.  yum install -y pulp-admin-client pulp-rpm-admin-extensions
```

Comment:
i'd probably err on the side of including none rather than including all, but the rpm one is the most useful.